testutil/direct_tx_gen.go

```diff
@@ -0,0 +1,272 @@
+package testutil
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/testutil/blobfactory"
+	blobtypes "github.com/celestiaorg/celestia-app/x/blob/types"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/stretchr/testify/require"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	coretypes "github.com/tendermint/tendermint/types"
+)
+
+// RandBlobTxsWithAccounts will create random blob transactions using the
+// provided configuration. The account info is queried directly from the
+// application. One blob transaction is generated per account provided.
+func RandBlobTxsWithAccounts(
+	t *testing.T,
+	capp *app.App,
```

Comment:
We need to sign txs with valid sequences and account numbers, and have to query for those w/o grpc if we want to use the testapp, so I created even more testing functions (that we should consolidate in #1114) to query the app directly